types/tx/types.go

```diff
@@ -204,13 +199,5 @@ func ValidateMsg(msg sdk.Msg) error {
 		return err
 	}
 
-	signers := msg.GetSigners()
```

Comment:
With this PR, ValidateMsg == ValidateBasic. I propose to remove `ValidateMsg` altogether (it was introduced in #9418)

Reply:
[removed](https://github.com/cosmos/cosmos-sdk/pull/9915/commits/6eece0b20e5a929bea2ef5b2c977088ed4261ccc#diff-a9fd0fd3117eb07051f6554d5dcb4ebbaed74eda850795f62640f3646c3e4d10L193-L203)